include/cpuinfo.h

```diff
@@ -613,6 +613,12 @@ struct cpuinfo_cluster {
 #endif
 	/** Clock rate (non-Turbo) of the cores in the cluster, in Hz */
 	uint64_t frequency;
+	/** Is this a cluster of power_efficient/little cores?
+	 * For ARM/ARM64, with Linux or Mach: set to true if the first core in the
+	 *     least capable cluster is a little core
+	 * For others: set to false for now
+	 */
+	bool is_power_efficient;
```

Comment:
Should this be just name is_little_core? power efficient is arguable. Big core can be power efficient too.

Reply:
Going with ARM terminology - https://www.arm.com/why-arm/technologies/big-little
This bool will be false for Big cores (based on MIDR) even if they are actually power efficient e.g. clocked lower or something like that.